src/windows/window-snapper.js

```diff
@@ -1,40 +1,52 @@
-let getEdges = bounds => {
-  return {
-    top: bounds.y,
-    bottom: bounds.y + bounds.height,
-    left: bounds.x,
-    right: bounds.x + bounds.width
-  }
-}
-
-let isCloseTo = (a, b, snapThreshold) => {
-  return Math.abs(a - b) <= snapThreshold
-}
-
-module.exports = (windowBounds, screenBounds, snapThreshold) => {
+const snapCheck = (windowBounds, screenBounds, snapThreshold) => {
   if (snapThreshold <= 0) {
-    return { x: windowBounds.x, y: windowBounds.y }
+    throw new Error('Not supported, not supposed to call window-snapper if threshold <= 0!')

```

Comment:
When does this happen? Is it a programmatic error or a user error?

Reply:
This is a programmatic error, the code calling this module has a check for `snapThreshold` 0, so this will only occur if that code changes, and the developer will get quick feedback (fail fast). Does that make sense? 😅